doc/fidget.md

```diff
@@ -146,6 +151,25 @@ regularly overlaid on top of buffer text (which is supported but unsightly).
 
 Type: `bool` (default: `true`)
 
+#### window.relative
```

Comment:
Nit: complete sentences with a period.

Reply:
Let me know if I fixed the right one.

Reply:
Missed the plurality of your comment, will fix all of them.